### ISSUE TITLE
CAP Shields With Big Energy Weapons

### ISSUE THREAD

**Issue by Matspyder51** [MattJeanes/PewPew#35]
I have create new ships or my server, and one of this ship is the Hat'ak, so his shoots had a size/damages of 30 and when I shoot on the shield, the Ressource node or E2PZ is automaticly destroy. I have comment the stargateshield plugins file, but I don't know very much the pewpew and I don't know how can I block this damages ?

Thank you and Sorry for my English

**Comment by Divran** [comment]
The stargate shield compatibility has been disabled because it doesn't work

**Comment by Matspyder51** [comment]
Yeah, I have comment the file, but if I shoot on the shield with the Hat'ak, the ressource node or the ZPM explode

**Comment by MattJeanes** [comment]
I believe you misunderstood. The stargate shield code in PewPew is disabled because it is broken. Whether it will be fixed, I do not know.
